src/goto-programs/safety_checker.h

```diff
@@ -30,7 +30,20 @@ class safety_checkert:public messaget
     const namespacet &_ns,
     message_handlert &_message_handler);
 
-  enum class resultt { SAFE, UNSAFE, ERROR };
+  enum class resultt
+  {
+    SAFE,
```

Comment:
Might as well document the existing ones while you're there!

Reply:
(done)